### PR TITLE
docs: Correct grammar, punctuation, add link

### PR DESCRIPTION
**Pull request by vivganes**
docs: Correct grammar, punctuation, add link

Rewrote some sentences to make the 'flow' better.  Added a link to text 'Configuration File'.

### CODE REVIEW

**Comment by vivganes** [comment]
I signed it!

File: docs/config/03-browsers.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-Capturing browsers on your own can be a tedious and time-consuming task,
-so Karma can automate this for you. Simply add the browsers you would like to
-capture into the configuration file:
+Capturing browsers on your own can be a tedious and time-consuming task.However, Karma can automate this for you. Simply add the browsers you would like to capture into the [configuration file].
```
Comment (dignifiedquire) [inline docs/config/03-browsers.md]: missing space `task.However`

**Comment by vivganes** [comment]
@dignifiedquire  I have added another commit to the branch.  Now I have added a space.  Thanks for pointing out the error.

**Comment by dignifiedquire** [comment]
Thanks :octocat:

**Pull request merged**